apps/src/code-studio/pd/teachercon1819_registration/Teachercon1819Registration.jsx

```diff
@@ -13,9 +13,10 @@ import { TeacherSeatAcceptanceOptions } from '@cdo/apps/generated/pd/teachercon1
 export default class Teachercon1819Registration extends FormController {
   static propTypes = {
     ...FormController.propTypes,
-    applicationId: PropTypes.number.isRequired,
+    applicationId: PropTypes.number,
+    regionalPartnerId: PropTypes.number,
     applicationType: PropTypes.string.isRequired,
-    course: PropTypes.string.isRequired,
+    course: PropTypes.string,
```

Comment:
Is `course` no longer required?

Reply:
it isn't; it's now only used by the teacher and facilitator versions